oauthproxy.go

```diff
@@ -319,6 +323,7 @@ func (p *OAuthProxy) buildProxySubrouter(s *mux.Router) {
 	s.Path(signOutPath).HandlerFunc(p.SignOut)
 	s.Path(oauthStartPath).HandlerFunc(p.OAuthStart)
 	s.Path(oauthCallbackPath).HandlerFunc(p.OAuthCallback)
+	s.Path(clearUserSessions).HandlerFunc(p.ClearUserSessions)
```

Comment:
If for any reason this registration causes problems, we can get rid of it. I think it's fine but the more important one is the registration above (line 302) for internal use.

Reply:
@joel-bluedata removed this registration